boot/misc/generate_evdev_data.rb

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env nix-shell
+#!nix-shell --packages linuxHeaders ruby -i ruby
+
+# This script can be used to update the input_event_codes ruby file.
+# The file
```

Comment:
I think you are missing something here.

Reply:
Thanks! I must have had my thoughts interrupted :)